src/main/java/org/broadinstitute/hellbender/tools/exome/eval/EvaluateCopyNumberTriStateCalls.java

```diff
@@ -436,22 +441,28 @@ private VariantEvaluationContext composeNonTruthOverlappingVariantContext(final
 
     private Genotype composeNonTruthOverlappingGenotype(final VariantContext enclosingContext, final Genotype genotype) {
         final GenotypeBuilder builder = new GenotypeBuilder(genotype.getSampleName());
-        GATKProtectedVariantContextUtils.setGenotypeQualityFromPLs(builder, genotype);
-
-        final int[] PL = genotype.getPL();
-        final int callAlleleIndex = GATKProtectedMathUtils.minIndex(PL);
-        final double quality = callQuality(genotype);
-        builder.alleles(Collections.singletonList(enclosingContext.getAlleles().get(callAlleleIndex)));
-        builder.attribute(VariantEvaluationContext.CALL_QUALITY_KEY, quality);
-        final boolean discovered = XHMMSegmentGenotyper.DISCOVERY_TRUE.equals(
-                GATKProtectedVariantContextUtils.getAttributeAsString(genotype, XHMMSegmentGenotyper.DISCOVERY_KEY,
-                        XHMMSegmentGenotyper.DISCOVERY_FALSE));
-        if (callAlleleIndex != 0 && discovered) {
-            builder.attribute(VariantEvaluationContext.EVALUATION_CLASS_KEY, EvaluationClass.UNKNOWN_POSITIVE.acronym);
-        }
-        if (quality < filterArguments.minimumCalledSegmentQuality) {
-            builder.filter(EvaluationFilter.LowQuality.acronym);
-        } else {
+        if (genotype.isCalled()) {
+            GATKProtectedVariantContextUtils.setGenotypeQualityFromPLs(builder, genotype);
+            final int[] PL = genotype.getPL();
+            final int callAlleleIndex = GATKProtectedMathUtils.minIndex(PL);
+            final double quality = callQuality(genotype);
+            builder.alleles(Collections.singletonList(enclosingContext.getAlleles().get(callAlleleIndex)));
+            builder.attribute(VariantEvaluationContext.CALL_QUALITY_KEY, quality);
+            final boolean discovered = XHMMSegmentGenotyper.DISCOVERY_TRUE.equals(
+                    GATKProtectedVariantContextUtils.getAttributeAsString(genotype, XHMMSegmentGenotyper.DISCOVERY_KEY,
+                            XHMMSegmentGenotyper.DISCOVERY_FALSE));
+            if (callAlleleIndex != 0 && discovered) {
+                builder.attribute(VariantEvaluationContext.EVALUATION_CLASS_KEY, EvaluationClass.UNKNOWN_POSITIVE.acronym);
+            }
+            if (quality < filterArguments.minimumCalledSegmentQuality) {
+                builder.filter(EvaluationFilter.LowQuality.acronym);
+            } else {
+                builder.filter(EvaluationFilter.PASS);
+            }
+        } else { /* assume it is neutral */
+            /* TODO this is a hack to make CODEX vcf work */
```

Comment:
Maybe a little more detail about why this hack is necessary and what will be done about it (if anything).  Create an issue if necessary.